src/App.js

```diff
@@ -31,6 +33,7 @@ function App() {
           </Route>
         </Switch>
       </StyledApp>
+      <ToastContainer position="bottom-right" autoClose={5000} pauseOnHover/>
```

Comment:
I think autoClose=5000 and pauseOnHover are the defaults, you can leave them if you think it's more readable though